### DISCUSSION TITLE
-Batch not (yet?) available for Set-PnPListItem

### DISCUSSION THREAD

**Discussion by PHubaut** [pnp/powershell#312] [Q&A] [answered]
I am a little bit confused about current status of "batching". According to the "[Batching in PnP PowerShell](https://pnp.github.io/powershell/articles/batching.html)", I was expecting the `-Batch` parameter available on these 3 Cmdlets: `Add-PnPListItem`, `Remove-PnPListItem` and `Set-PnPListItem`.
However, while testing, the two first Cmdlets worked like a charm but the latter failled as `-Batch` parameter is not recognized.
I tried with both the `v1.3.0` stable version and the `v1.3.9-nightly` but results are the same (PSVersion: `5.1.19041.610`)
Did I miss some implementation schedule, or should I report this as an issue? Thanks for your advises.

**Comment by veronicageek** [comment]
@erwinvanhunen - I don't see it either on the `Set-PnPListItem` (PS7).

**Comment by veronicageek** [comment, marked as answer]
@PHubaut - This has been fixed following issue #327.

**Reply by PHubaut** [reply]
Thanks a lot !
